### ISSUE TITLE
xcode8 compatibility

### ISSUE THREAD

**Issue by 4ndrey** [goktugyil/EZSwiftExtensions#192]
your pod is not usable under xcode8, please support new swift updates.

**Comment by 515783034** [comment]
please support new swift updates.

**Comment by odedharth** [comment]
is the branch Swift3 ready?

**Comment by Luccifer** [comment]
Looks like nope... At least 60+ errors by compiling any type of branches

**Comment by goktugyil** [comment]
https://github.com/goktugyil/EZSwiftExtensions/issues/191

I won't have access to a computer for a week. Feel free to send pull requests about the errors. I imagine they would be simple syntax changes, with a few exceptions on some serious changes in uikit framework. 

If you guys don't want to go through all that trouble, I think there is way to specify something like "Use Swift2.2 compiler for this file/framework". That may be a simple fix until.

lfarah can approve your pull requests and EZSwift can start working properly again very soon.

**Comment by adhiravishankar** [comment]
#193

**Comment by adhiravishankar** [comment]
#194

**Comment by lfarah** [comment]
Guys, #194 isn't working right now.

I took some methods off and updated to Swift 3 [in my fork](https://github.com/lfarah/EZSwiftExtensions)

You can use on Cocoapods doing the following:
``` ruby
  pod 'EZSwiftExtensions', :git => 'https://github.com/lfarah/EZSwiftExtensions.git'
```

**Comment by lfarah** [comment]
Let me know if you guys run into any problems!

**Comment by piv199** [comment]
Swift 3 branch should work.

**Comment by goktugyil** [comment]
So there has been few people who converted this to Swift3 without pulling from the latest version. As a result a lot of the new features/methods/tests have disappeared.

I'm going to reconvert the whole thing starting from #190 in a separate branch, then merge it in master. This shouldn't affect the master branch or people who has their projects working.

**Comment by piv199** [comment]
@goktugyil I've done it in #198 it was based on #190, fixed for swift 3.0 and merged with #194 then.

**Comment by goktugyil** [comment]
Sorry for the delay, just released a stable version for Swift 3.0

```ruby
platform :ios, '8.0'
use_frameworks!
pod 'EZSwiftExtensions' #Stable release for Swift 3.0

pod 'EZSwiftExtensions', :git => 'https://github.com/goktugyil/EZSwiftExtensions.git' #Latest release for Swift 3.0
pod 'EZSwiftExtensions', :git => 'https://github.com/goktugyil/EZSwiftExtensions.git', :branch => 'Swift2.3' #For Swift 2.3
pod 'EZSwiftExtensions', '~> 1.6' #For Swift 2.2
```

**Comment by Luccifer** [comment]
You are the best!
